### ISSUE TITLE
None of my names are showing in "Manage Names"

### ISSUE THREAD

**Issue by csharpner** [namecoin/namecoin-core#253]
After purchasing 120+ names, none of them show up under "Manage Names" in the Namecoin Core wallet GUI app on Windows.  The transactions are in the transactions tab though.

I expected all of my purchased names to be there.

![namecoin blank](https://user-images.githubusercontent.com/8125269/43997758-cac848d0-9db2-11e8-93b6-332738e70b98.png)

Open the app.  Click on the "Manage Names" tab.  Nothing there.

I'm not using Bitcoin Core.  I'm using Namecoin Core.  It's version:  nc0.13.99-name-tab-beta1 (64 bit)

Windows 10 Pro N, latest updates.
CPU:  Intel Core i5-4690K 3.5Ghz (quadcore)
16GB RAM
Boot drive = 512GB SSD
Data Drive = 12TB RAID 3 Tower with four 4TB drives.  App is installed on that drive.

When I purchased each name, it was presented with a dialog box to configure it, which I clicked cancel on, because I don't have sites ready for it.
Also, while purchasing, after the first 27, it falsely reported that I was out of funds, though I had plenty.  After stopping it and restarting several times (might have had a reboot in there), I was able to continue the next day and got those false funds errors again.  After stopping and restarting the app a couple times, I was able to continue.  Now, I have no names listed in my "Manage Names" list.  Before they disappeared, I noticed that many that I did purchase were not showing up there.

**Comment by JeremyRand** [comment]
The "insufficient funds" error happens when you try to build a chain of unconfirmed transactions that's too long, which can occur if you try to create a large number of name transactions at once.  The correct way to resolve it is to wait for some of your transactions to be mined, not to restart Namecoin Core.  (IIRC the error message for this is more clear in more recent Namecoin Core versions.)

I'm wondering whether maybe the frequent restarting of Namecoin Core while the `name_firstupdate` transactions were pending might have caused them to not get broadcast.  @domob1812 , am I correct in remembering that the txid and salt for each `name_new` will get saved to `debug.log`?  If that's correct, and the `name_firstupdate` transactions have erroneously not been broadcast, would the correct resolution be to check `debug.log` and manually run `name_firstupdate` from the console?

**Comment by domob1812** [comment]
Yes, those are saved to `debug.log` - my intent in doing so was exactly to allow rescuing those registrations in case the return value from `name_new` has been lost somehow.
